jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceSummaryErrorPopup.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<!--
+  ~ Copyright 2017 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<div class="panel panel-default" style="background-color: #fff; padding: 17px 32px; font-weight: bold">
```

Comment:
There is quite a bit of css style rules in here, you can probably move these into a specific css file. You can check an Errai UI css binding example in here: https://github.com/kiegroup/jbpm-wb/blob/d5989ce04cdc88a2c376b3e681e79c8bb14b04cc/jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/list/CaseInstanceListViewImpl.java#L39